src/services/bug/controller.ts

```diff
@@ -0,0 +1,35 @@
+import { StatusCodes } from 'http-status-codes';
+
+import { FastifyPluginAsync } from 'fastify';
+
+import { BUGS_RECIVER_EMAIL } from '../../utils/config';
+import { bugSchema } from './fluent-schema';
+import { generateEmail } from './utils';
+
+const plugin: FastifyPluginAsync = async (fastify) => {
+  fastify.post<{ Body: { name: string; email: string; details: string } }>(
+    '/',
+    {
+      schema: bugSchema,
+      preHandler: fastify.verifyAuthentication,
+    },
+    async (request, reply) => {
+      const { member, body: data } = request;
+      const { log, mailer } = fastify;
+
+      const subject = `Graasp Builder | User Feedback`;
+      const html = generateEmail(data);
```

Comment:
There're actually some util functions to generate an email, it would be better to use them I think https://github.com/graasp/graasp/blob/main/src/plugins/mailer/index.ts